cpp/locality/LinkCell.cc

```diff
@@ -299,11 +299,6 @@ const vec3<unsigned int> LinkCell::computeDimensions(const box::Box& box, float
 
 void LinkCell::computeCellList(const vec3<float>* points, unsigned int n_points)
 {
-    if (n_points == 0)
```

Comment:
@yuanzhou0827 I'm confused about why this edit from #592 is showing in the diff. Maybe a branch wasn't merged correctly?